ambry-rest/src/main/java/com/github/ambry/rest/PublicAccessLogHandler.java

```diff
@@ -28,8 +28,12 @@
 import java.util.Collection;
 import javax.net.ssl.SSLEngine;
 import javax.security.auth.x500.X500Principal;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+//import org.slf4j.Logger;
```

Comment:
nit: lets remove these two lines instead of commenting them out.

Reply:
OK that is done.